BUILD

```diff
@@ -39,6 +39,8 @@ pkg_tar(
         "//plaidbench:wheel",
         "//plaidml:wheel",
         "//plaidml/bridge/keras:wheel",
+        "//plaidml/edsl/tests:cc_test",
```

Comment:
This approach requires that support for all runtimes is linked into `//plaidml/edsl/tests:cc_test` at all times; I was under the impression that we didn't want to do that, but I could be mistaken.  :-)

Reply:
Eventually, I plan to have multiple `cc_test` targets, but for now this works just fine.

Reply:
For tests I actually do want everything to be built and linked, so that even if someone is on a machine that can't run the tests, they can at least build them, so that people have less chance of breaking some other platform. I think the whole idea of transitions in bazel isn't a great one.

Reply:
SGTM